mitodl/mitxpro
# Limit full name length to 255 characters

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #2575 

#### What's this PR do?
Limits `User.name` length to 255 characters and truncates any longer than that during the migration (there is only 1 on production, and that name is just spam and maybe the user should be deactivated).

#### How should this be manually tested?
On the master branch, register as a new user or edit the profile of an existing user, providing a full name over 255 characters long.
Switch to this brsnch, restart containers.  Check the long-named user, the name should now be truncated.
Register as a new user.  Try to give a full name > 255 characters, the form should invalidate it and display an error until you shorten it.


#### Screenshots (if appropriate)
![Screenshot 2023-02-16 at 4 12 35 PM](https://user-images.githubusercontent.com/187676/219489362-47107e95-5c95-4674-918b-4e50c76c741b.png)
